CAIRN Info.js

```diff
@@ -84,6 +84,22 @@ function doWeb(doc,url)
 						tags.push(keywords[i])
 					}
 				}
+				
+				// The default value for PDF download is on an HTML page that calls the actual download
+				// we need to add this attachment after the import translator has been running
+				// We remove the attached PDF because they won't be working
+				for(var i=0, n=item.attachments.length; i<n; i++) {
+					if(item.attachments[i].mimeType == 'application/pdf') {
+						delete item.attachments[i];
```

Comment:
@aurimasv I have tried to mix your suggestion and @adam3smith's one to fix this part, not sure it was what you were thinking of, otherwise let me know